thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js

```diff
@@ -155,6 +179,12 @@ export default Ember.Component.extend({
     }
   ),
 
+  /**
+   * Compute changes from a given offset to the current time range, as a fraction.
+   *
+   * @param {String} offset time range offset, e.g. 'baseline', 'wow', 'wo2w', ...
+   * @returns {Object} change values, keyed by metric urn
+   */
```

Comment:
Woohooo! I love comments. Thanks for this and also `@param` and `@return`. Quick note: `@return`, not `@returns` 😉

Reply:
auto-generated by intellij, both are valid: http://usejsdoc.org/

Reply:
Ah, gotcha. Didn't realize that. Good to know!